app/javascript/components/App.js

```diff
@@ -29,8 +30,37 @@ const App = (props) => {
   }
 
   const createApartment = (apartment) => {
-    console.log(apartment)
-  }
+        fetch("http://localhost:3000/apartments", {
+          method: "POST", 
+          headers: {
+            "Content-Type": "application/json", 
+          },
+          body: JSON.stringify({ apartment }),
+        })
+        .then((response) => response.json())     
+        .then(() => {
+          setApartments(apartment)
+          console.log("Created Apartment:", apartment)
+        })
```

Comment:
This part right here may not be needed since we are submitting the apartment directly to the database, we do not need to update state with it directly, we can update state by calling on our readApartments to pull the updated database information.